backend/src/main/java/capstone/storage/backend/item/models/Item.java

```diff
@@ -0,0 +1,11 @@
+package capstone.storage.backend.item.models;
+
+public record Item(String id,
```

Comment:
Der Name `Item` passt aus meiner Sicht nicht perfekt: Objekte dieser Klasse speichern ja auch Lagermenge, etc. stimmt's?
Vorschlag:
```suggestion
public record Stock(String id,
```

Reply:
die Klasse Item verwaltet nicht die Menge an Artikeln die ich einlagern möchte ,die Klasse Item verwaltet quasi.. ist nicht schön aber alles andere würde den Ramen sprengen wie viele Artikel ich von diesem Typen Item einlagern kann. Ein Artikel bzw Item ist immer einmalig und soll nur beschreiben welche Artikel ich generell in meinem Lager einlagern kann mit der Beschreibung wie viele davon auf ein Lagerplatz passen